source/docs/casper/concepts/index.md

```diff
@@ -4,6 +4,7 @@
 - [Understanding Hash Types](./understanding-hash-types.md)
 - [Understanding Call Stacks](./callstack.md)
 - [Contracts and Session Code](./session-code.md)
+- [Understanding Global State](./global-state.md)
```

Comment:
According to sidebar order, global state should appear before callstack and session code:
```suggestion
- [Understanding Global State](./global-state.md)
- [Understanding Call Stacks](./callstack.md)
- [Contracts and Session Code](./session-code.md)
```